burr/core/application.py

```diff
@@ -1395,6 +1401,9 @@ def initialize_from(
         :param resume_at_next_action: Whether to resume at the next action, or default to the ``default_entrypoint``
         :param default_state: The default state to use if it does not exist. This is a dictionary.
         :param default_entrypoint: The default entry point to use if it does not exist or you elect not to resume_at_next_action.
+        :param fork_from_app_id: The app ID to fork from. This is used to fork from a prior application run.
```

Comment:
Worth mentioning how these differ from `with_identifiers` -- this is a bit confusing